@here/olp-sdk-dataservice-read/lib/client/OlpClientSettings.ts

```diff
@@ -60,6 +60,12 @@ export interface OlpClientSettingsParams {
      * @see interface [[DownloadManager]]
      */
     dm?: DownloadManager;
+
+    /**
+     * Billing Tag is an optional free-form tag which is used for grouping billing records together.
+     * If supplied, it must be between 4 - 16 characters, contain only alpha/numeric ASCII characters [A-Za-z0-9].
+     */
+    billingTag?: string;
```

Comment:
Definitely not in Settings.
This is per request! Settings can be shared between clients.
This needs to be added to each request class: CatalogRequest, DataRequest, PartitionsRequest, etc. as an optional parameters.

Reply:
Updated. Added to each request. Thank you